timemachine/cpp/src/local_md_potentials.hpp

```diff
@@ -68,6 +69,9 @@ class LocalMDPotentials {
     DeviceBuffer<float> d_probability_buffer_;
 
     DeviceBuffer<unsigned int> d_free_idxs_;
+    DeviceBuffer<unsigned int>
+        d_all_pairs_idxs_;                  // Where intersection of free/frozen and initial atom indices are stored
```

Comment:
nit: The naming here is unexpected, d_all_pairs_idxs_ is what I'd think the 'initial atom indices, of NonbondedAllPairs, are stored'

Reply:
Changed in https://github.com/proteneer/timemachine/pull/1038/commits/7dbeabcffb9e5b0cc94b1ccebbf69ebaf8eefb1a